grid/address/geocoding/maps/__snapshots__/azure-maps.test.ts.snap

```diff
@@ -38,12 +38,10 @@ Ok {
 
 exports[`address/geocoding/azure-maps ReverseGeocode when inputs are invalid should throw an exception 1`] = `
 Err {
-  "error": "MappedHTTPError: Expected HTTP error
-Properties: {
-  \\"title\\": \\"Bad request\\",
-  \\"detail\\": \\"The provided coordinates in query are invalid, out of range, or not in the expected format\\"
-}
-AST Path: definitions[1].statements[1].responseHandlers[1].statements[0]
-Original Map Location: Line 130, column 7",
+  "error": "InputValidationError: Input validation failed:
```

Comment:
Here we could also send input as `{ latitude: undefined, longitude: undeifned }` as we would get the same result as before.

Which would pass [required field](https://superface.ai/docs/comlink/profile#sec-Required-field) validation and thus the request would be send.